Drive/rover_drive_system.hpp

```diff
@@ -380,8 +424,8 @@ class RoverDriveSystem
   };
 
   char current_mode_   = 'S';
-  int state_of_charge_ = 90;  // TODO - hardcoded for now
-
+  int state_of_charge_ = static_cast<int>(StateOfCharge_MAX());  // TODO - hardcoded for now
+  int state_of_charge_LTC = static_cast<int>(StateOfCharge_LTC());
```

Comment:
We use the Google Style Guide for C++. Class member variables should end with and underscore e.g. `state_of_charge_LTC_`